molecule/default/converge.yml

```diff
@@ -5,11 +5,13 @@
 
   pre_tasks:
     - name: Update apt cache.
-      apt: update_cache=yes cache_valid_time=600
+      ansible.builtin.apt:
+        update_cache: yes
```

Comment:
Might as well set update_cache to "true" for _thruthyness_ as elsewhere in the role.